types/src/validator_verifier.rs

```diff
@@ -188,10 +187,7 @@ impl<PublicKey: VerifyingKey> ValidatorVerifier<PublicKey> {
         // Fallback is required to identify the source of the problem if batching fails.
         if PublicKey::batch_verify_signatures(&hash, keys_and_signatures).is_err() {
             match self.verify_aggregated_signature(hash, aggregated_signature) {
-                Ok(_) => warn!(
-                    "Inconsistency between batch and iterative signature verification detected! \
-                     Batch verification failed, while iterative passed."
-                ),
+                Ok(_) => { /* nothing */ },
```

Comment:
Is this actually ok? Instead of warning I think it would be better to return an error instead.